### DISCUSSION TITLE
Need for HTTPS for Admin Interface when connecting via VPN?

### DISCUSSION THREAD

**Discussion by SorenK49** [AdguardTeam/AdGuardHome#4110] [Q&A]
Hello,

I have set up AdGuard Home on a laptop, which works flawlessly and covers my home network.

In order to access it remotely, I can connect to the admin web interface (and the AdGuardHome iOS app) via Tailscale which uses Wireguard protocol. Tailscale also allows me to use AGH to resolve DNS on my phone when I’m outside of my home network. 

My question is, given this connection between my phone and laptop is encrypted, is there any need to make the admin web interface HTTPS?

I am not very tecinically savvy so I am not sure whether there is any benefit in changing it from HTTP to HTTPS as I am not connecting to it via a Domain but rather through a VPN tunnel. My understanding is that I also cannot get a certificate without registering a domain, which I don’t need to do in this case as I connect via Tailscale.

I hope this question makes sense.

Thanks for developing such a great product.

**Comment by ainar-g** [comment]
Apologies for the late response.  I'd say, if you don't intend using DNS-over-TLS and other secure protocols, and you reasonably trust your VPN provider, it's probably not necessary.
